apps/scan/frontend/src/components/signed_hash_validation_button.tsx

```diff
@@ -0,0 +1,54 @@
+import React, { useState } from 'react';
```

Comment:
Should these buttons be factored out to a common component?

Reply:
Yes good q! I'm working on exactly that (and other changes) as part of my next PR